src/status_im2/contexts/activity_center/notification/common/view.cljs

```diff
@@ -35,13 +35,14 @@
            interpolation-translate-x
            on-press
            swipe-button
-           swipeable-ref]}]
+           swipeable-ref
+           style]}]
```

Comment:
hmm do we need style as a param here or can it just be statically imported from the style file?